.ci/run_e2e_tests.sh

```diff
@@ -13,16 +13,17 @@ DOCKER_REPO="avaplatform"
 AVALANCHE_TESTING_REPO=$DOCKER_REPO/avalanche-testing
 DEFAULT_TEST_SUITE_IMAGE="$AVALANCHE_TESTING_REPO:dev"
 BYZANTINE_IMAGE="$DOCKER_REPO/avalanche-byzantine:v0.1.5-rc.1"
+CI_COMMIT_TAG="avalanchego-ci-${COMMIT}"
```

Comment:
Oh I see, you wanted it to be clear that the commit-tag suffix is from the ci run. Makes sense